pymilvus/orm/iterator.py

```diff
@@ -195,13 +210,15 @@ def __init__(
         **kwargs,
     ) -> SearchIterator:
         if len(data) > 1:
-            raise MilvusException(message="Not support multiple vector iterator at present")
+            raise ParamError(message="Not support multiple vector iterator at present")
```

Comment:
Not clear enough

"Not support search iteration over multiple vectors at present"

Reply:
changed